models/model.js

```diff
@@ -107,3 +107,17 @@ exports.selectUsers = () => {
     return users.rows
   })
 }
+
+exports.removeComment = (comment_id) => {
+  return db 
+  .query(`
+  DELETE FROM comments
+  WHERE comment_id = $1
+  RETURNING *;
```

Comment:
Don't need RETURNING * here - you can look on the result object to see if there's anything else of use to determine if the comment existed or not